sdk/python/cmd/pulumi-language-python-exec

```diff
@@ -68,7 +68,18 @@ if __name__ == "__main__":
         os.chdir(args.pwd)
 
     successful = False
-    loop = asyncio.get_event_loop()
+
+    # asyncio.get_running_loop was only added in python 3.7 but we still support python 3.6
+    # In python 3.10, asyncio.get_event_loop prints a deprecation warning if no loop is present
+    # This code will be cleaned up as part of https://github.com/pulumi/pulumi/issues/8131
+    if sys.version_info[0] == 3 and sys.version_info[1] < 7:
+        loop = asyncio.get_event_loop()
+    else:
+        try:
+            loop = asyncio.get_running_loop()
```

Comment:
The [docs for `get_running_loop`](https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.get_running_loop) state that this function can only be called from a coroutine or a callback. Are we in one of those contexts here?

Reply:
I don't think we ever would be but I wasn't trying to make any assumptions and merely mirror the functionality of `asyncio.get_event_loop()` in a manner that cooperates with python 3.10. I would think just these two lines would be sufficient:

Reply:
I'm gonna go ahead and just do what I said in the comment above.